### DISCUSSION TITLE
video download helper not working

### DISCUSSION THREAD

**Discussion by 123456wertyudfgh** [aclap-dev/video-downloadhelper#310] [Help and Q&A]
### Your question

hello

i am using the latest google chrome and encounted this issue which does not matter which site it just wont download please fix the issue

kind regards

![Screenshot 2024-03-19 063548](https://github.com/aclap-dev/video-downloadhelper/assets/154477560/c68e7771-c0ec-49aa-8f4d-612c070403f8)

### Addon & CoApp version and details

hello

i am using the latest google chrome and encounted this issue which does not matter which site it just wont download please fix the issue

kind regards

![Screenshot 2024-03-19 063548](https://github.com/aclap-dev/video-downloadhelper/assets/154477560/c68e7771-c0ec-49aa-8f4d-612c070403f8)

**Comment by paulrouget** [comment]
See this https://github.com/aclap-dev/video-downloadhelper/discussions/308#discussioncomment-8834832
